abb_driver/rapid/ROS_socket.sys

```diff
@@ -29,7 +29,7 @@ MODULE ROS_socket(SYSMODULE)
 
 PROC ROS_init_socket(VAR socketdev server_socket, num port)
     IF (SocketGetStatus(server_socket) = SOCKET_CLOSED) SocketCreate server_socket;
-    IF (SocketGetStatus(server_socket) = SOCKET_CREATED) SocketBind server_socket, GetSysInfo(\LanIp), port;
+    IF (SocketGetStatus(server_socket) = SOCKET_CREATED) SocketBind server_socket, "192.168.1.42", port;

```

Comment:
This is a known hack we have to make things work in our hardware setup, but will be removed for the final PR.

Reply:
There should be no need. If you have your controller setup properly (ie: not using any internal interfaces, which is why @yamokosk needs this), `GetSysInfo(\LanIp)` actually returns the correct IP.

Binding on all interfaces *could* work (not sure whether RW likes that), but I'm not a fan.